khmer/khmer_args.py

```diff
@@ -389,18 +393,22 @@ def add_loadgraph_args(parser):
                         help='load a precomputed k-mer graph from disk')
 
 
-def calculate_graphsize(args, graphtype, multiplier=1.0):
+def calculate_graphsize(args, graphtype, multiplier=1.0, _sysmem=None):
```

Comment:
If it causes practical problems, we'll eliminate it.  Hard to know at this
point. But memory settings are one of the major causes of user pain,
at least anecdotally.